scripts/build

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 set -e
 
-composer install --dev
\ No newline at end of file
+rm -f composer.lock
```

Comment:
What's the purpose of this? It seems like we would not want to blow this away unless we are explicitly wishing to upgrade the packages.